src/test/ui/argument-suggestions/invalid_arguments.stderr

```diff
@@ -0,0 +1,349 @@
+error[E0059]: arguments to this function are incorrect
+  --> $DIR/invalid_arguments.rs:13:11
+   |
+LL |   one_arg(1.0);
+   |           ^^^ expected i32, found {float}
+   |
+note: function defined here
+  --> $DIR/invalid_arguments.rs:5:4
+   |
+LL | fn one_arg(_a: i32) {}
+   |    ^^^^^^^ -------
+
+error[E0059]: multiple arguments to this function are incorrect
+  --> $DIR/invalid_arguments.rs:16:19
+   |
+LL |   two_arg_same(1, "");
+   |                   ^^ ^ missing argument of type i32
+   |                   |
+   |                   no parameter of type &'static str is needed in two_arg_same
+   |
+note: function defined here
+  --> $DIR/invalid_arguments.rs:6:4
+   |
+LL | fn two_arg_same(_a: i32, _b: i32) {}
+   |    ^^^^^^^^^^^^ -------  -------
+help: the following changes might help
+   |
+LL |   two_arg_same(1, ) {i32},;
+   |                  -- ^^^^^^
+
+error[E0059]: multiple arguments to this function are incorrect
+  --> $DIR/invalid_arguments.rs:17:16
+   |
+LL |   two_arg_same("", 1);
+   |                ^^    ^ missing argument of type i32
+   |                |
+   |                no parameter of type &'static str is needed in two_arg_same
+   |
+note: function defined here
+  --> $DIR/invalid_arguments.rs:6:4
+   |
+LL | fn two_arg_same(_a: i32, _b: i32) {}
+   |    ^^^^^^^^^^^^ -------  -------
+help: the following changes might help
+   |
+LL |   two_arg_same(1) {i32},;
```

Comment:
This is suggesting

```rust
two_arg_same(1) {i32},;
```

but shouldn't it be suggesting

```rust
two_arg_same(1, {i32},);
```

or something similar? Diff:

```diff
- two_arg_same(1) {i32},;
+ two_arg_same(1, {i32},);

```